types/host.go

```diff
@@ -28,7 +31,7 @@ type Host interface {
 	Memory() (*HostMemoryInfo, error)
 
 	// FQDN returns the fully-qualified domain name of the host, lowercased.
-	FQDN() (string, error)
```

Comment:
This is a breaking change to the API. I suggest adding an `FQDNWithContext(context.Context)` unless we are prepared to rollout a `github.com/elastic/go-sysinfo/v2`.

Reply:
Done in 8ef547d.